ftw/contacts/resources/contactfolder_listing.js

```diff
@@ -0,0 +1,81 @@
+$(function() {
+    ContactFolderListing.init();
+});
+var ContactFolderListing = (function($){
+    "use strict"
+    var self = {};
+    var $contactsContainer;
+    var $loadMoreButton;
+    var $alphabeticalContainer;
+    var $searchInput;
```

Comment:
It's actually a bad habit to use `$` as prefix, as it is in fact reserved for the JavaScript interpreter as internal global variables. We should get rid of this habit. Could you rename all the variables to not have a prefixing `$`?